content/typescript/concepts/narrowing/narrowing.md

```diff
@@ -22,7 +22,7 @@ TypeScript recognizes types as being narrowed when it sees branches of JavaScrip
 
 - `if/else` conditional statements.
 - Other conditionals, like ternaries and switches.
-- Looping statments, such as `for`- and `while`-loops.
+- Looping statements, such as `for`- and `while`-loops.
 - Truthiness checks for entities that don't point to `false`-like values.
```

Comment:
```suggestion
- Conditional statements like `if/else`, ternaries, and switches.
- Looping statements, such as `for`- and `while`-loops.
- Truthiness checks for entities that don't point to `false`-like values.
```

Reply:
@garanews Any chance we could implement this suggestion? The first two list items seem inherently related, hence why I recommended consolidating into one item about conditional statements.

Reply:
@Dusch4593 was not able to commit your suggestion, had to edit by my side, should be ok now.